src/internal/item-container/index.tsx

```diff
@@ -239,7 +250,7 @@ function ItemContainerComponent(
     }
 
     // Notify the respective droppable of the intention to insert the item in it.
-    draggableApi.acquire(nextDroppable);
+    draggableApi.acquire(nextDroppable, acquiredItemElement);
```

Comment:
The `acquiredItemElement` is the same as `children()`. Can we pass the `childrenRef.current` here instead?

Reply:
In that case there is also no need to pass the `acquiredItemElement` to the item container.